wizard/callbacks.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2025 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package wizard
+
+import (
+	"errors"
+	"os"
+
+	"github.com/sourcenetwork/defradb/crypto"
+	"github.com/sourcenetwork/defradb/keyring"
+)
+
+// This callback will set keyring.backend to either "file" or "system"
+func callback_SetKeyringBackend(s step, ctx *WizardContext) error {
+	mm := s.(*modelMultipleChoice) //nolint:forcetypeassert
```

Comment:
question: why do we need the `nolint` here?

suggestion:
```suggestion
	mm, ok := s.(*modelMultipleChoice)
	if !ok {
		return errHandleHere
	}
```